src/network/ConnectionRegistry.ts

```diff
@@ -66,6 +66,12 @@ export interface ConnectionRegistry {
      * @return Error if invocation is not allowed, null otherwise
      */
     checkIfInvocationAllowed(): Error | null;
+
+    /**
+     * Returns {@code true} if the client is initialized on the cluster, by
```

Comment:
```suggestion
     * Returns `true` if the client is initialized on the cluster, by
```

We generally use backticks. Not sure about `@code`

Reply:
Fixed